crates/forge/tests/integration/setup_fork.rs

```diff
@@ -0,0 +1,123 @@
+use crate::integration::common::corelib::{corelib_path, predeployed_contracts};
+use crate::integration::common::running_tests::run_test_case;
+use crate::{assert_passed, test_case};
+use camino::Utf8PathBuf;
+use dotenv::dotenv;
+use forge::scarb::{ForgeConfig, PredefinedFork};
+use forge::{run, RunnerConfig};
+use indoc::formatdoc;
+use std::collections::HashMap;
+
+#[test]
+fn fork_simple_decorator() {
+    dotenv().ok();
```

Comment:
Shouldn't `unwrap()` be used here instead so it panics on failed dotenv load?

Reply:
Actually true, we should do that in 
```rust
#[cfg(test)]
#[ctor::ctor]
fn init() {
```
tooo